build_library/check_root

```diff
@@ -177,9 +177,20 @@ def check_shebang():
                 line = fd.readline(80)
                 if not line.startswith(b"#!"):
                     continue
-                cmd = line[2:].rstrip().split(None,1)[0]
+                words = line[2:].rstrip().split(None, 1)
```

Comment:
As far as I know, Linux doesn't support multiple arguments on the `#!` line.  I can still update it anyway, for portability (Container Illumos?).